apps/arc/arc.talon

```diff
@@ -0,0 +1,14 @@
+app: arc
+-
+tag(): browser
+tag(): user.tabs
+
+profile switch: user.arc_mod("shift-m")
+
+tab search: user.arc_mod("shift-a")
```

Comment:
This also doesn't work in Arc. You can achieve something similar using cmd-t, although that is more similar to the command palette in vscode. Maybe you can replace this with a similar command to `please` in vscode.

Reply:
This is a great idea!! I've added a please command and I like it a lot. Thank you so much for the suggestion.